src/main/java/mate/jdbc/sevice/DriverServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package mate.jdbc.sevice;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id).get();
+    }

```

Comment:
What if you get null? I think you should throw exception here.

Reply:
agree